### ISSUE TITLE
Please, release the last tags into cocoapods!

### ISSUE THREAD

**Issue by danielborges93** [jpotts18/SwiftValidator#172]
We have a lib that depends you and we can't run codes in swift3. We depend the tag 4.0.0.

**Comment by davepatterson** [comment]
Have you tried using `pod 'SwiftValidator', :git => 'https://github.com/jpotts18/SwiftValidator.git', :branch => 'master'`?

**Comment by danielborges93** [comment]
Yes, but it do not solve the problem. I use a library that have the dependency `s.dependency 'SwiftValidator'` and when I run `pod install`, cocoapods always downloads the version 3.0.3 (that is [the last on the cocoapods spec](https://cocoapods.org/pods/SwiftValidator)).

**Comment by danielborges93** [comment]
OBS.: My code is in Swift3.

**Comment by davepatterson** [comment]
So what is it that you're asking me? To create a new tag that mirrors master, which is written in Swift3? Because I definitely can't change tag 4.0.0 to be compatible with Swift3.

**Comment by danielborges93** [comment]
No, I am asking to release the tag 4.x into cocoapods. [You have a lot of releases after 3.0.3](https://github.com/jpotts18/SwiftValidator/releases), but the [last cocoapods release is the 3.0.3](https://cocoapods.org/pods/SwiftValidator).

I've tried to force cocoapods to download the branch `master` in my Podfile (`pod 'SwiftValidator', :git => 'https://github.com/jpotts18/SwiftValidator.git', :branch => 'master'`), but other dependences that depend you make cocoapods download the version 3.0.3 (that is the last in cocoapods).

**Comment by smamczak** [comment]
Please do it, I have the same problem. I'm using SwiftValidator as pod lib where I cannot set branch, only tag version as you can see here https://guides.cocoapods.org/syntax/podspec.html#dependency

**Comment by jpotts18** [comment]
I'm not going to lie this project took off and I'm pretty overwhelmed by it. I no longer actively code in Swift and I need help from more contributors in order to keep this project relevant. I'd like to find someone to transition as lead maintainer or help this codebase find a happy home with a company.

The codebase for this project is relatively small and focused project for who is looking to contribute. It does one thing, and does it pretty well. It has pretty high test coverage and good adoption. I think my ideal maintainer / contributor would work for a company that actively uses SwiftValidator in their production application.

If you're interested in helping out please comment on #204
